packages/core-mobile/app/services/network/NetworkService.ts

```diff
@@ -132,14 +132,18 @@ class NetworkService {
     return ModuleManager.avalancheModule.getProvider(mapToVmNetwork(network))
   }
 
-  private async fetchERC20Networks(): Promise<Networks> {
-    const response = await fetch(`${Config.PROXY_URL}/networks`)
-    const networks: Network[] = await response.json()
+  private async fetchPrimaryNetworks(includeSolana: boolean = false): Promise<Networks> {
```

Comment:
```suggestion
  private async fetchNetworks(includeSolana: boolean = false): Promise<Networks> {
```

The term “primary networks” in our team usually refers only to the Avalanche C/X/P Chains, Ethereum, and Bitcoin networks.